src/components/svg/index.tsx

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove this line since it is not needed.

Reply:
It was for some reason, error with stencil